Classes/Emogrifier.php

```diff
@@ -430,7 +428,11 @@ protected function process(\DOMDocument $xmlDocument)
             $this->removeInvisibleNodes($xPath);
         }
 
+        $this->removeImportantAnnotationFromAllInlineStyles($xPath);
+
         $this->copyCssWithMediaToStyleNode($xmlDocument, $xPath, $cssParts['media']);
+
+        restore_error_handler();
```

Comment:
Moved `restore_error_handler()` here, because both `removeImportantAnnotationFromAllInlineStyles()` and `copyCssWithMediaToStyleNode()` make xPath queries.